src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AzureFunction/AzureFunction.csproj

```diff
@@ -13,6 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.0.1" />
```

Comment:
Did we confirm if this dependency is always available for Azure Function apps?

Reply:
Are we directly using anything from this reference?

Reply:
I'm using `HttpContext` in `FunctionsHttpProxyingMiddlewareWrapper` - I suppose it's not strictly required, but it sure simplifies things. I could use `dynamic` just as easily, I guess, if this reference is going to be an issue.

The package reference is an ancient version of the common abstractions, so I don't expect it should be a problem for us to reference it directly. I am correct that we'll use a newer version if the customer references in their app, right?

Reply:
... and switching to `dynamic` ended up being an easy change. Reference removed.